Formula/i686-elf-binutils.rb

```diff
@@ -16,6 +17,7 @@ class I686ElfBinutils < Formula
   def install
     system "./configure", "--target=i686-elf",
                           "--prefix=#{prefix}",
+                          "--libdir=#{lib}/i686-elf-binutils",
                           "--infodir=#{info}/i686-elf-binutils",
```

Comment:
I wonder if it's better to call these subdirectories `i686-elf`. Might be a better convention for the other cross-arch formulae. It would entail changing a few more formulae though.

Reply:
I chose what's really `<prefix>/#{name}` to be consistent with the rest of the formula, and also to automatically avoid name clashes in shared namespaces like `/share` and `/lib`.

That said, I'm not wedded to the idea, but if we keep this naming convention, I'll revise the formulae and use `#{name}` instead of hardcoding those directory component names.

Reply:
Hm, I'm not sure you understood what I was trying to say. I was suggesting that we use
```
--libdir=#{lib}/i686-elf
--infodir=#{info}/i686-elf
```
etc here and in the other `i686-elf-*` formulae. It seems more natural to have all the `i686-elf` share a single `lib` (or `info`, or others) directory in the prefix rather than to be split across different formulae (and similarly for all the `x86_64-elf-*`, etc).

Reply:
Ah, well, it's just a matter of adding three more formulae to this PR, so why not? :grinning: